config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -49,7 +49,7 @@ periodics:
         bucket: ppc64le-kubernetes
         path_strategy: explicit
       gcs_credentials_secret: gcs-credentials
-    interval: 3h
+    cron: "55 */3 * * *"
```

Comment:
But this will not put lot of stress on the pod where the tests are run right?

Reply:
Though `periodic-kubernetes-containerd-conformance-test-ppc64le` job runs tests against a PowerVS cluster, while the conformance suite is running, the to/fro calls from pod to cluster and other response verification on pod might cause some resource usage is what I was thinking.
As we are changing the schedule time of other periodic jobs, I thought to place this one also.